chain/mempool.go

```diff
@@ -15,7 +15,7 @@ import (
 
 // txNotFoundErr is an error returned from bitcoind's `getrawtransaction` RPC
 // when the requested txid cannot be found.
-const txNotFoundErr = "-5: No such mempool or blockchain Transaction"
+const txNotFoundErr = "-5: No such mempool or blockchain transaction"
```

Comment:
IIUC, we do a `ToLower()` in any case?

Reply:
Ah looks like not, perhaps we should add that for future proofing?

Reply:
you mean add a test?

Reply:
> you mean add a test?

That as well. But I think Laolu meant just doing `strings.ToLower()` on both strings before comparing them, to avoid such issues in the future.

Reply:
oh gotya!